test/fuzz/mmap_alloc.cc

```diff
@@ -0,0 +1,29 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ * Copyright (c) 2021-2022 Intel Corporation
+ */
+
+#include <cne_mmap.h>
+#include <string.h>
+/* Prototype required to fix "no previous prototype for function" error */
+extern "C" int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size);
+
+/* When launching the mmap_alloc fuzz test, the -rss_limit_mb=N parameter
+ * needs to be passed. Otherwise, an out of memory error will happen.
+ */
+extern "C" int
+LLVMFuzzerTestOneInput(const uint8_t *data, size_t size)
+{
+    uint32_t bufcnt, bufsz;
+    mmap_type_t hugepage;
+    mmap_t *mm;
+
+    if (size > sizeof(bufsz))
+        size = sizeof(bufsz);
+    memcpy(&bufcnt, data, size);
+    hugepage = (mmap_type_t)(bufcnt % MMAP_HUGEPAGE_CNT);
```

Comment:
Bufcnt could be some random value if size == 0, so setting it to zero at the top of the function is reasonable. If size is less than 4 then we could have some random value in the remaining bytes of the bufcnt variable, the reason to set it to zero up front.